### ISSUE TITLE
Github releases and Nuget update?

### ISSUE THREAD

**Issue by echoix** [dallasbeek/Dapper.Database#36]
I see that the package published on Nuget is dated from 2020, but there was some work done here in 2021. There are also no tags or releases in here that may indicate that the package is usable in a new project.

Does it still work with current releases of Dapper, and .net?

**Comment by dallasbeek** [comment]
Yes, I'm using the Nuget package with the latest versions of Dapper and .net core 6.  I haven't published a nuget because appveyor changed a bunch of stuff under me and I haven't had time to put it back together.

**Comment by echoix** [comment]
I tried locally to make it open in VS2022, and adding .net 6.0 in the targeted frameworks, but I didn't really manage to make everything compile. I installed way more optional VS features so I don't know what worked, and the .netstandard2.0 appeared often in the errors. I had to remove the globale.json (rename to *.old) to make it compile (without 6.0). 

I noticed that the sourcelink dependency does not exist anymore, but is now part of dotnet (I'm using it currently for my project).

I don't have much (recent) experience in .NET, especially with multitargeting. I'm not fully aware of the history expect like .NET Framework 4.5-4.6 many many years ago.
